src/el/geth/geth_launcher.star

```diff
@@ -99,11 +99,16 @@ def launch(
 
     network_name = (
         "devnets"
-        if launcher.network != "kurtosis"
-        and launcher.network != "ephemery"
+        if launcher.network != constants.NETWORK_NAME.kurtosis
+        and launcher.network != constants.NETWORK_NAME.ephemery
+        and launcher.network != constants.NETWORK_NAME.shadowfork
         and launcher.network not in constants.PUBLIC_NETWORKS
         else launcher.network
     )
+    if (
+        constants.NETWORK_NAME.shadowfork in network_name
+    ):  # if its a shadowfork we want proper resource allocation based on public network
+        network_name = launcher.network.split("-shadowfork")[0]
```

Comment:
i wonder if we can pull out network name splitting into its own method

Reply:
Done